config/application-FILEIO.properties

```diff
@@ -8,5 +8,5 @@
 #io.putGradeIO=./configputGrades-V-7170-E.txt
 
 #io.getGradeIO=./config/getGrades-TEST-EMPTY.json
-io.getGradeIO=./config/getGrades-TEST-A.json
-io.putGradeIO=./config/putGrades-V-7170-E.txt
+#io.getGradeIO=./config/getGrades-TEST-A.json
```

Comment:
now that all lines are commented out in this file, do we need keep the file itself?

Reply:
The file is an example of how to configure SPE to bypass the ESB for both input and output.  It is a bit hard to explain so having an example file is useful.